src/sentry/features/__init__.py

```diff
@@ -101,6 +101,7 @@
 default_manager.add("organizations:issue-search-use-cdc-primary", OrganizationFeature, FeatureHandlerStrategy.REMOTE)
 default_manager.add("organizations:issue-search-use-cdc-secondary", OrganizationFeature, FeatureHandlerStrategy.REMOTE)
 default_manager.add("organizations:issue-search-group-attributes-side-query", OrganizationFeature, FeatureHandlerStrategy.REMOTE)
+default_manager.add("organizations:issue-severity-alerts", OrganizationFeature, FeatureHandlerStrategy.REMOTE)
```

Comment:
@malwilley as long as this flag isn't used during post process then we can keep using flagr

Reply:
Yeah we'll use an option there